app/src/main/java/com/sun/makeupwindow/ui/home/HomeFragment.kt

```diff
@@ -9,15 +11,16 @@ import com.sun.makeupwindow.data.repository.ProductRepository
 import com.sun.makeupwindow.data.source.ProductDataSource
 import com.sun.makeupwindow.ui.dialog.LoadingDialog
 import com.sun.makeupwindow.ui.productdetail.ProductDetailFragment
-import com.sun.makeupwindow.utlis.RepositoryFactory
-import com.sun.makeupwindow.utlis.addFragment
-import com.sun.makeupwindow.utlis.showToast
+import com.sun.makeupwindow.utlis.*
 import kotlinx.android.synthetic.main.fragment_home.*
 
 class HomeFragment : BaseFragment(), HomeContract.View {
 
     private var presenter: HomePresenter? = null
     private var loadingDialog: LoadingDialog? = null
+    private var loadMoreItems = mutableListOf<Product>()
+    lateinit var scrollListener: RecyclerViewLoadMoreScroll
+    lateinit var mLayoutManager: RecyclerView.LayoutManager
```

Comment:
```suggestion
    private lateinit var layoutManager: RecyclerView.LayoutManager
```
mà mấy cái trên k để private được à